canonicaljson.py

```diff
@@ -95,7 +115,7 @@ def iterencode_canonical_json(json_object):
         yield chunk.encode("utf-8")
 
 
-def encode_pretty_printed_json(json_object):
+def encode_pretty_printed_json(json_object: object) -> bytes:
```

Comment:
Should we update the docstrings to remove the types?

Reply:
Yes. I was being lazy.

The docstrings all suggest that the encoding functions are designed to take dictionaries, but the tests have examples of encoding other things e.g. one character strings:

https://github.com/matrix-org/python-canonicaljson/blob/16a9335cd1a8a0f6b6fe72655fbf2291caf5f53f/test_canonicaljson.py#L87-L88

Reply:
Done in https://github.com/matrix-org/python-canonicaljson/pull/49/commits/ba77219a287330185cf6a7650c6e410c330fe11d.